src/Symfony/Component/DependencyInjection/Definition.php

```diff
@@ -49,6 +49,13 @@ class Definition
 
     private static $defaultDeprecationTemplate = 'The "%service_id%" service is deprecated. You should stop using it, as it will soon be removed.';
 
+    /**
+     * @internal
+     *
+     * Used to store the name of the renamed id when using service decoration together with autowiring
+     */
+    public $previousRenamedId;
```

Comment:
The name of this property is confusing to me: `previousRenamedId`  "previous" and "renamed" together is confusing: is this the previous ID before the renaming? Is this the new renamed ID different from the previous one? Is this the first renamed ID and now we are doing the second renaming (so this is the "previous rename"? etc.

Reply:
I think it's a rest of the previous implementation (where `renamedId` was already used). is ~previous~`renamedId` ok to you?

Reply:
It depends on what this property really is (sorry, but its PHPdoc is confusing to me too). Here are some alternative proposals:

* `$decoratedServiceOriginalId`
* `$decoratedServiceNewId`
* `$decoratedServiceId`
* ...

Reply:
I agree the name is confusing

Reply:
Done (I picked `innerServiceId`). Is it good for you guys?

Reply:
Much better. Thanks!